html2docx/image.py

```diff
@@ -0,0 +1,91 @@
+import io
+import pathlib
+import time
+import urllib.error
+import urllib.request
+from typing import Dict, Optional
+
+from docx.image.exceptions import UnrecognizedImageError
+from docx.image.image import Image
+from docx.shared import Inches
+
+USABLE_HEIGHT = Inches(8.1)
```

Comment:
Where do these numbers come from? Are they something that can be reference or just empirical? If there is a reference, can it be here in a comment?

Reply:
In the commit message, there is an explanation:

> The usable area was determined by resizing an image in LibreOffice in the default template document. The maximum height for an image in a document is capped to 8.1 inches.

Do you think it would be better as a comment?

Reply:
Thanks, I skipped over the commit message, I should have read that.

I do think including a comment here as well would be beneficial as this may look like a magic number to new contributors or when coming back to this in a year.